frameworks/g-cloud-9/questions/services/lotName.yml

```diff
@@ -0,0 +1,2 @@
+question: 'Product type'
+type: text
```

Comment:
I'm 99% sure I know... all previous G-Cloud frameworks have it and I'm pretty sure it is a non-editable placeholder that is used to display the name of the lot that a service belongs to on the page.  e.g. pages for suppliers or admin editing a service, and maybe even the G-0Cloud service pages.  The service data gets updated with a `lotName` field, and then the manifest used to display the service can refer to the `lotName` "question" to show it on the page.